doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
 Depends xxx.
 
 <a name="6.3"></a>
-## 6.3 Descriptor file definition
+## 6.3 Descriptor file definition specification
 There must be a Descriptor File definition, which used by installer as input of necessary configuration.
-Mandatory and optional definition shall be defined.
+Mandatory and optional definition shall be defined.  
+All the required definition for description file is listed in this specification session, there's no restrictions on how to use it,
```

Comment:
"this specification session" is unclear to me ... does it mean "this release"?

Reply:
I was refering "the current section", but I realize this sentence is unneccesary, so I removed it